drivers/hud_path_report/app/controllers/hud_path_report/cells_controller.rb

```diff
@@ -0,0 +1,35 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudPathReport
+  class CellsController < HudPathReport::QuestionsController
+    before_action :set_report
+    before_action :set_question
+
+    def show
+      @cell = @report.valid_cell_name(params[:id])
+      @table = @report.valid_table_name(params[:table])
+      @clients = HudPathReport::Fy2020::DqClient.
```

Comment:
should this be `DqClient`?  Maybe PathClient?

Reply:
Good catch!